src/icons/README.md

```diff
@@ -0,0 +1,27 @@
+# Icons setup
+
+## Principle
+A list of file names and content is generated using the `generateIconsMap.js` node script, which fetches all the svg files in the `svg-icons` directory here. This generates a SASS file `_generated-icons-map.scss` that holds a SASS map variable, which in turn is used to generate icon css classes like this; `.icon-[file-name]` with property `background-image: url([encoded svg content]);`.
+
+### Source files
```

Comment:
`### Source files` -> `## SVG source files`
Can't really see that this is a sub to Principle... and maybe ok to add the most important info to the title?